tests/lean_spec/subspecs/ssz/test_signed_attestation.py

```diff
@@ -21,10 +27,14 @@ def test_encode_decode_signed_attestation_roundtrip() -> None:
                 source=Checkpoint(root=Bytes32.zero(), slot=0),
             ),
         ),
-        signature=Signature.zero(),
+        signature=Signature(
+            path=HashTreeOpening(siblings=HashDigestList(data=[])),
+            rho=Randomness(data=[Fp(0) for _ in range(PROD_CONFIG.RAND_LEN_FE)]),
+            hashes=HashDigestList(data=[]),
+        ),
     )
 
+    # Test that encoding and decoding round-trips correctly
     encode = signed_attestation.encode_bytes()
-    expected_value = "0" * 6504
-    assert encode.hex() == expected_value
-    assert SignedAttestation.decode_bytes(encode) == signed_attestation
+    decoded = SignedAttestation.decode_bytes(encode)
+    assert decoded == signed_attestation

```

Comment:
shouldn't we keep an assert against a hardcoded answer here. Just because we want other clients to implement these tests and make sure they get the same results, as the spec does